helper.py

```diff
@@ -52,19 +63,24 @@ def load_reader(
     sentinel_1_tif_folder = f"{competition}_{train_or_test}_source_sentinel_1"
     sentinel_2_tif_folder = f"{competition}_{train_or_test}_source_sentinel_2"
     planet_5day_tif_folder = f"{competition}_{train_or_test}_source_planet_5day"
-    s1_input_dir = f"{root}/{sentinel_1_tif_folder}/{sentinel_1_tif_folder}{fill}_asc_{pos}_2017"
-    s2_input_dir = f"{root}/{sentinel_2_tif_folder}/{sentinel_2_tif_folder}{fill}_{pos}_2017"
-    planet_5day_input_dir = f"{root}/{planet_5day_tif_folder}"
+    planet_daily_tif_folder = f"{competition}_{train_or_test}_source_planet"
+    s1_input_dir = str(
+        root / f"{sentinel_1_tif_folder}/{sentinel_1_tif_folder}{fill}_asc_{pos}_2017"
+    )
+    s2_input_dir = str(root / f"{sentinel_2_tif_folder}/{sentinel_2_tif_folder}{fill}_{pos}_2017")
```

Comment:
FYI the year is different for Germany dataset.  I had it updated in my recent commit so just keep it as is for now and I'll show you the changes in helper.py in the upcoming PR.